compiler-rt/lib/mpk_untrusted/mpk_formatter.cpp

```diff
@@ -90,7 +90,7 @@ void writeJSON(std::ofstream &OS,
 
 // Writes output of the faultSet to a uniquely generated output file to ensure
 // we do not overwrite previously discovered faulting values.
-bool writeUniqueFile(std::set<AllocSite> &faultSet) {
+bool writeUniqueFile(std::unordered_set<AllocSite, AllocSite::Hasher> &faultSet) {
```

Comment:
you may want to typedef or a using directive to make this a more convenient type
i.e., ``` using FaultSet std::unordered_set<AllocSite, AllocSite::Hasher>```

Reply:
Changed method of hashing AllocSite which simplifies these types.